gen_cert.py

```diff
@@ -85,14 +85,20 @@ def get_cert_date(
     - If using the "ROLLING" certs feature, use today's date
     - If all else fails use 'configured_date_parameter' for date
     """
+    utc_zone = get_timezone('UTC')
+    to_zone = get_timezone(timezone)
 
     if calling_date_parameter:
-        date_value = format_datetime(calling_date_parameter, 'MMMM d, y', tzinfo=timezone, locale=locale)
+        date = calling_date_parameter
     elif configured_date_parameter == "ROLLING":
-        date_value = format_datetime(datetime.datetime.today(), 'MMMM d, y', tzinfo=timezone, locale=locale)
+        date = datetime.today()
     else:
-        date_value = format_datetime(configured_date_parameter, 'MMMM d, y', tzinfo=timezone, locale=locale)
+        date = datetime.strptime(configured_date_parameter, '%y-%m-%d')
 
+    date = date.replace(tzinfo=utc_zone)
```

Comment:
strip off any timezone attached to the initial date object, cause the server should only deal in UTC